src/12-nextgen-projects/projects.apib

```diff
@@ -82,12 +89,18 @@ Lists all projects that match the optional filters provided.
                     + unit: `seconds` (enum[string])
                         + Members
                             + seconds
+                + `time_estimated` (object, nullable) - Combined value of all time estimated by lines linked to project, value in seconds, rounded to the nearest minute.
+                    + value: 43200 (number)
+                    + unit: `seconds` (enum[string])
+                        + Members
+                            + seconds
                 + `time_tracked` (object, nullable)
                     + `value`: 43200 (number)
                     + `unit`: `seconds` (enum[string])
                         + Members
                             + `seconds`
                 + `amount_billed` (Money, nullable)
+                + `amount_unbilled` (Money, nullable) - Price of the project minus the amount billed
```

Comment:
B: This is currently only available for internal requests

Reply:
We have already released this one :)